nodemcu/init.lua

```diff
@@ -1,17 +1,38 @@
 local wifimodule = require 'wifimodule'
 local socketmodule = require 'socketmodule'
+local config = require 'config'
 
-function init()
-  -- Initializes LED-strip
-  ws2812.init()
-  local i, buffer = 0, ws2812.newBuffer(8, 3)
+-- Initialize LED-strip and Timer
+ws2812.init()
+local i, buffer = 0, ws2812.newBuffer(8, 3)
+local ledTimer = tmr.create()
+
+-- Make the LED-strip loop
+function ledLoop(interval, colorArray)
+  ledTimer:register(interval, 1, function()
+    i = i + 1
+      buffer:fade(2)
+      buffer:set(i % buffer:size() + 1, colorArray[1], colorArray[2], colorArray[3])
+      ws2812.write(buffer)
+  end)
+
+  ledTimer:start()
+end
+
+function resetStrip()
+  buffer:fill(0, 0, 0)
+  ws2812.write(buffer)
+  ledTimer:stop()
+end
 
+-- Initialize application on NodeMCU
+function init()
   -- Receive socket events
   local ws = socketmodule.initSocket()
 
   ws:on('receive', function(_, msg)
     local data = cjson.decode(msg)
-    if data.type == 'color' then
+    if data.type == 'color' and data.id == node.chipid() then
```

Comment:
`data.type` is not send anymore. It has been changed to `data.action` with `CHANGE_COLOR`